api/provider/aws/service_list.go

```diff
@@ -0,0 +1,170 @@
+package aws
+
+import (
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (s *ServiceProvider) List() ([]models.ServiceSummary, error) {
```

Comment:
is this what was already in seshi's branch? if so, let's merge anyway for now. but still will need to be revisited.

Reply:
Yeah - our two branches seemed to be a little intertwined, but with Seshi being a new dad, I expect to take responsibility for all of service, so I can continue his work going forward.